backend/Api/Common/StorageService.cs

```diff
@@ -75,102 +92,95 @@ private List<Consultant> LoadConsultantsFromDb(string orgUrlKey)
         return hydratedConsultants;
     }
 
-    public void UpdateStaffing(int id, double hours)
+    private Core.DomainModels.Staffing CreateStaffing(StaffingKey staffingKey, double hours )
     {
-        var staffing = _dbContext.Staffing
-            .Include(s=>s.Project)
-            .ThenInclude(p=> p.Customer)
-            .ThenInclude(c=> c.Organization)
-            .Include(s=>s.Consultant)
-            .FirstOrDefault(staffing => staffing.Id == id);
-        if (staffing is null) return;
-        var orgUrlKey = staffing.Project.Customer.Organization.UrlKey;
-        staffing.Hours = hours;
-        _dbContext.SaveChanges();
-        var consultantId = staffing.Consultant.Id;
-        var consultants = LoadConsultants(orgUrlKey);
-        consultants
-            .Single(c => c.Id == consultantId)
-            .Staffings
-            .Single(s => s.Id == id)
-            .Hours = hours;
-        _cache.Set($"{ConsultantCacheKey}/{orgUrlKey}", consultants);
-    }
-
-    public void UpdateAbsence(int id, double hours)
-    {
-        var absence = _dbContext.PlannedAbsence
-            .Include(pa => pa.Absence)
-            .ThenInclude(a => a.Organization)
-            .Include(pa => pa.Consultant)
-            .FirstOrDefault(absence => absence.Id == id);
-        if (absence is null) return;
-        var orgUrlKey = absence.Absence.Organization.UrlKey;
-        absence.Hours = hours;
-        _dbContext.SaveChanges();
-        var consultantId = absence.Consultant.Id;
-        var consultants = LoadConsultants(orgUrlKey);
-        consultants
-            .Single(c => c.Id == consultantId)
-            .PlannedAbsences
-            .Single(pa => pa.Id == id)
-            .Hours = hours;
-        _cache.Set($"{ConsultantCacheKey}/{orgUrlKey}", consultants);
-    }
-
-    public int CreateStaffing(int consultantId, int projectId, double hours, Week week)
-    {
-        var consultant = _dbContext.Consultant.Find(consultantId);
+        var consultant = _dbContext.Consultant.First(c => c.Id == staffingKey.ConsultantId);
```

Comment:
```suggestion
        var consultant = _dbContext.Consultant.SingleOrDefault(c => c.Id == staffingKey.ConsultantId);
```
Dette _bør_ krasje om vi har to konsulenter med samme ID

Reply:
Krajser vel og om konsulenten ikke eksisterer, som den bør